hassio-hostapd/run.sh

```diff
@@ -104,6 +105,15 @@ echo "channel=${CHANNEL}" >> ${HCONFIG}
 echo "interface=${INTERFACE}" >> ${HCONFIG}
 echo "" >> ${HCONFIG}
 
+if test ${HIDE_SSID} = true;
+then
+    echo "Hidding SSID"
+    sed -i 's/ignore_broadcast_ssid=0/ignore_broadcast_ssid=1/' ${HCONFIG}
```

Comment:
Why is `sed` required?
Cant you just `echo` it in like the others?

Reply:
`sed` cause the `ignore_broadcast_ssid=0` already exist in the hostapd.conf